extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js

```diff
@@ -36,30 +44,54 @@ class AmpServiceWorkerInstallation extends AMP.BaseElement {
     const src = this.element.getAttribute('src');
```

Comment:
Assert `src` value itself? `assertHttpsUrl` will fallback to resolve against the current URL for empty input URL.